enterprise/signals.py

```diff
@@ -424,3 +424,14 @@ def generate_default_orchestration_record_display_name(sender, instance, **kwarg
 
 if COURSE_UNENROLLMENT_COMPLETED is not None:
     COURSE_UNENROLLMENT_COMPLETED.connect(enterprise_unenrollment_receiver)
+
+
+@receiver(pre_save, sender=models.EnterpriseCustomer)
+def copy_data_from_career_engagement_network_message(sender, instance, **kwargs):  # pylint: disable=unused-argument
+    """
+    Copy data from `career_engagement_network_message` and `enable_career_engagement_network_on_learner_portal` to
+    `learner_portal_sidebar_content` and `enable_learner_portal_sidebar_message` respectively.
+    """
+    enable_cen = instance.enable_career_engagement_network_on_learner_portal
```

Comment:
Isn't this will start giving errors in the next build when we will remove `enable_career_engagement_network_on_learner_portal` field?

Reply:
I don't think so because we will be removing all the references to the old fields including this signal in next release.